.vale/templates/bot-comment-output.tmpl

```diff
@@ -1,18 +1,49 @@
-{{- $githubCommitID := "githubID" -}}
-{{- range .Files }}
-  {{- $filePath := .Path }}
-  {{- range .Alerts }}
-    {{- $severity := "" -}}
-    {{- if eq .Severity "error" -}}
-        {{- $severity = "error" -}}
-    {{- else if eq .Severity "warning" -}}
-        {{- $severity = "warning" -}}
-    {{- else -}}
-        {{- $severity = "suggestion" -}}
-    {{- end -}}
-
-    {{- $loc := printf "%d:%d" .Line (index .Span 0) -}}
-    {{- $body := printf "{\"body\":\"%s\",\"path\":\"%s\",\"line\":%d}" .Message $filePath .Line }}
-    {{- $body }}
-  {{- end }}
-{{- end }}
\ No newline at end of file
+{{- /* Modify Vale's output https://docs.errata.ai/vale/cli#--output */ -}}
+
+{{- /* Keep track of our various counts */ -}}
+
+{{- $e := 0 -}}
+{{- $w := 0 -}}
+{{- $s := 0 -}}
+
+{{- /* Range over the linted files */ -}}
+
+[
+{{- range $jdx, $file := .Files -}}
+
+{{- $path := .Path -}}
+
+{{- /* Range over the file's alerts */ -}}
+
+{{- if $jdx -}},{{- end -}}
+{{- range $idx, $a := .Alerts -}}
+
+{{- $error := "" -}}
+{{- if eq .Severity "error" -}}
+    {{- $error = "error" -}}
+    {{- $e = add1 $e  -}}
+{{- else if eq .Severity "warning" -}}
+    {{- $error = "warning" -}}
+    {{- $w = add1 $w -}}
+{{- else -}}
+    {{- $error = "suggestion" -}}
+    {{- $s = add1 $s -}}
+{{- end}}
+
+{{- /* Variables setup */ -}}
+
+{{- $loc := printf "%d" .Line -}}
+{{- $check := printf "%s" .Check -}}
+{{- $message := printf "%s" .Message -}}
+{{- if $idx -}},{{- end -}}
+
+{{- /* Output */ -}}
+
+  {
+    "description": "[{{ $error }}] {{$check}}: {{ $message }}",
```

Comment:
@aireilly - the github review comment API uses the body key. Is it worth keeping that consistent here to avoid confusion? 
`{"body":"'"$BODY"'","commit_id":"'"$COMMIT_ID"'"`

Reply:
updated